### PR TITLE
Solve #541

### PR DESCRIPTION
**Pull request by byronsliger**
Also:
Changing name locator to accessibilityId locator, in order to execute the actions with 'by.name' in device, because it was missing.
And Updating setPlaform because 'capabilities.platform' was empty.

### CODE REVIEW

**Comment by Astro03** [comment]
@jlipps @penguinho  who created the checks above?  can we have someone fix this?

File: Appium/Inspector/AppiumInspectorWindowController.m [hunk 1]
```diff
@@ -53,6 +53,10 @@ - (id)initWithWindow:(NSWindow *)window
             [capabilities addCapabilityForKey:@"automationName" andValue:(model.isAndroid ? model.android.automationName : @"Appium")];
             [capabilities addCapabilityForKey:@"platformName" andValue:(model.isAndroid ? model.android.platformName : @"iOS")];
 			[capabilities addCapabilityForKey:@"platformVersion" andValue:model.isAndroid ? model.android.platformVersionNumber : model.iOS.platformVersion];
+			[capabilities addCapabilityForKey:@"newCommandTimeout" andValue:@"999999"];
```
Comment (jlipps) [inline Appium/Inspector/AppiumInspectorWindowController.m]: the value of 0 means no newCommandTimeout

**Comment by jlipps** [comment]
@Astro03 I don't understand your question

**Comment by Astro03** [comment]
There is a check when there is a pull request. Assuming it's trying to build.  It's failing.  Was wondering how I can set it up to build the new target. 

Ciao,
Astro
Sent from my phone

On May 6, 2016, at 22:14, Jonathan Lipps <notifications@github.com> wrote:

@Astro03 I don't understand your question

**Comment by penguinho** [comment]
@Astro03 , should be in the travis.yml I believe

**Comment by Astro03** [comment]
Perfect.  I'll fix it when I get back into the states on the 25th.  

Ciao,
Astro
Sent from my phone

On May 9, 2016, at 15:38, Dan Cuellar <notifications@github.com> wrote:

@Astro03 , should be in the travis.yml I believe

**Comment by edgy360** [comment]
I might have beaten you to it @Astro03! 😄

**Comment by Astro03** [comment]
Thanks man.  I'm Currently on travels in asia and do not have a computer on me.  

Thanks a bunch. 

Ciao,
Astro
Sent from my phone

On May 11, 2016, at 00:55, Jamie Edge <notifications@github.com> wrote:

I might have beaten you to it @Astro03! 😄

**Comment by edgy360** [comment]
No problem! Have a good trip! :)

**Pull request closed without merging**